### PR TITLE
Car Classifier Example

### PR DESCRIPTION
**Pull request by mecevit**

### CODE REVIEW

**Review by volkangurel — APPROVED** [review]

File: car_classifier/data/features/car_features/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+torch==1.7.1
+torchvision==0.8.2
+numpy
```
Comment (kocabiyikalper) [inline car_classifier/data/features/car_features/requirements.txt:3]: seems like we need `pandas` instead

File: car_classifier/data/features/car_parts_features/backlight_feature/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+torch==1.7.1
+torchvision==0.8.2
+numpy
```
Comment (kocabiyikalper) [inline car_classifier/data/features/car_parts_features/backlight_feature/requirements.txt:3]: `pandas` ^

File: car_classifier/data/features/car_parts_features/backlight_feature/requirements.txt [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+Pillow
+pycocotools
```
Comment (kocabiyikalper) [inline car_classifier/data/features/car_parts_features/backlight_feature/requirements.txt:5]: Where do we need this package?

File: car_classifier/data/features/car_parts_features/dataset.yml [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+description: "Parts of the car as image"
+
+features:
+  - name: backlight
+    description: "Backlight of the car"
+    source: backlight_feature/feature.py
+    environment: backlight_feature/requirements.txt
+  - name: year
+    description: "Build year of the car"
+    source: build_year.sql
```
Comment (kocabiyikalper) [inline car_classifier/data/features/car_parts_features/dataset.yml:15]: I am not sure if we support mixed type featuresets -> https://github.com/The-Layer/monorepo/blob/8e24f95f64cc3668731ba967a6faac6dba3d4b2e/sdk/tests/e2e/test_project.py#L342

File: car_classifier/models/car_classifier/requirements.txt [hunk 5]
```diff
@@ -0,0 +1,5 @@
+torch==1.7.1
+torchvision==0.8.2
+numpy
+Pillow
+sklearn
```
Comment (kocabiyikalper) [inline car_classifier/models/car_classifier/requirements.txt:5]: `pandas` ^

**Pull request closed without merging**